packages/jest-worker/src/index.ts

```diff
@@ -17,6 +17,7 @@ import type {
   WorkerPoolInterface,
   WorkerPoolOptions,
 } from './types';
+export {default as PriorityQueue} from './PriorityQueue';
```

Comment:
should we export `FifoQueue` as well? And the `TaskQueue` interface I guess so it's slightly easier to plug in your own?

Reply:
Exporting the `TaskQueue` interface is a great idea. 

I'm not sure what the value of exporting the `FifoQueue` is except. It can't be parametrized and is taken as the default. However, it could make the code more readable if some implementation either picks a priority or fifo queue (` x ? new PriorityQueue(...) : new FifoQueue()` vs `x ? new PriorityQueue(...) : undefined`.

Reply:
Yeah, that was the idea :+1: